openmdao/core/system.py

```diff
@@ -306,7 +306,7 @@ def _set_vars_as_remote(self):
             meta['remote'] = True
 
     def fd_jacobian(self, params, unknowns, resids, total_derivs=False,
```

Comment:
I am surprised that never caused a problem. I will set it to None in the other branch of the if. (This happens if we do fd on an unconnected input, so only in check_partial_derivatives)